Redis cluster support

hbasria opened the pull request:
Redis cluster support added

hbasria committed:
redis 3 cluster support added

hbasria committed:
cluster settings added

hbasria committed:
requirements file updated

fpieper commented:
Why are the checks failing? Just because the build is broken?
I am asking because I want to use it (directly from your fork hbasria).

snyk-bot committed:
fix: requirements-dev.txt to reduce vulnerabilities


The following vulnerabilities are fixed by pinning transitive dependencies:
- https://snyk.io/vuln/SNYK-PYTHON-SQLPARSE-6615674

hbasria committed:
Merge pull request #1 from hbasria/snyk-fix-a5cefea541cb2d63a52c8114956f8b0a

[Snyk] Security upgrade sqlparse from 0.4.4 to 0.5.0
